doc/rados/operations/crush-map.rst

```diff
@@ -965,18 +965,17 @@ default behavior, the ``ceph-mon`` and ``ceph-osd`` will prevent older
 clients who do not support the new CRUSH features from connecting to
 the cluster.
 
-argonaut (legacy)
+argonaut (CRUSH_TUNABLES or legacy)
```

Comment:
Here http://docs.ceph.com/docs/master/rados/operations/crush-map/#which-client-versions-support-crush-tunables , we can read

```
Which client versions support CRUSH_TUNABLES
* argonaut series, v0.48.1 or later1
```

So if CRUSH_TUNABLES is no equivalent to "legacy", what is the meaning of CRUSH_TUNABLES?